### PR TITLE
Alerts component for Appeals v2.

### PR DESCRIPTION
**Pull request by markgreenburg**
Components that together render a list of alerts - one for each object in the `appeal.attributes.alerts` array. Also introduces a helper function into `helpers-v2` that maps content to each alert type.

To-Do:
[x] - Wire up Alerts and Alert to each other, Status page
[x] - Finish logic for both components (prop passing, conditional styling based on alert type)
[x] - Make sure everything looks right at different viewport sizes
[x] - Everything accompanied by unit tests

**Full**
screen shot 2017-11-27 at 12 11 57 pm

**Mobile**
screen shot 2017-11-27 at 12 11 43 pm

### CODE REVIEW

**Comment by gnakm** [comment]
I wasn't fully aware of the new look for alert messages (My bads, I should have checked). I was hoping that the icons in the alert messages would align with the grey arrow. 

Example:
image

However, this is not going to work because it's going to break the grid on the left. 

Can we move the error message so that the highlighted bar is centered aligned with the arrow? 
Example:
image

image

**Comment by markgreenburg** [comment]
Excellent point @gnakm - I just updated the alignment to have the vertical bars match up with the Timeline - I updated the screenshots in the first comment.

**Comment by markgreenburg** [comment]
Sorry, looks like I didn't hit 'save' after uploading new screenshots - should be good now

**Review by cvalarida — COMMENTED** [review]
My only real concern is the alert object structure inference--I don't think it'll look quite like that. Should be a pretty quick fix.

The rest of it looks good, though. 👍

**Review by gnakm — APPROVED** [review]
lgtm! Thanks!

File: src/js/claims-status/containers/AppealsV2StatusPage.jsx [hunk 1]
```diff
@@ -49,9 +49,10 @@ AppealsV2StatusPage.defaultProps = {
     type: '',
     attributes: {
       events: [],
+      alerts: [],
       status: {
         type: '',
-        details: {}
+        details: {},
```
[the remark this answers is not in the document]
Comment (markgreenburg) [reply src/js/claims-status/containers/AppealsV2StatusPage.jsx]: No, just habit. Trying to break that habit for consistency with the rest of the team but sometimes I forget :)

File: src/js/claims-status/utils/helpers.js [hunk 2]
```diff
@@ -283,8 +283,18 @@ export const mockData = {
         ],
         alerts: [
           {
-            type: 'tbd', // Need to get a real status type
-            details: {}
+            type: 'warning',
+            date: '09-21-2017',
+            details: {
+              type: 'waiting_on_action'
```
[the remark this answers is not in the document]
Comment (markgreenburg) [reply src/js/claims-status/utils/helpers.js]: Yeah, I bet you're right. Chris G mentioned that shape will be consistent with other parts of the API, so I'm assuming it'll look like Status does. I've simplified this and re-worked a bit, though we may need to tweak again once the API is live.

File: src/js/claims-status/utils/appeals-v2-helpers.jsx [hunk 3]
```diff
@@ -246,3 +258,61 @@ export function getNextEvents(currentStatus) {
...
+  switch (details.type) {
+    case ALERT_CONTENT_TYPES.waitingOnAction:
+      return {
+        title: 'Your appeal is waiting on action by your representative',
+        description: `Your appeal is near the front of the line, but it is not
+          ready to go to a judge. It is currently waiting on your
+          representative, the ${details.representative}, to complete an
+          informal hearing presentation (IHP). Please contact your
+          representative for more information.`,
+        cssClass: 'usa-alert-warning',
```
[the remark this answers is not in the document]
Comment (markgreenburg) [reply src/js/claims-status/utils/appeals-v2-helpers.jsx]: I'm just going to hardcode a `cssClass` property value for each `alert.type` for now and get rid of the STYLE_TYPES enum for now. We'll see if this needs to change once we can use the API.

**Comment by markgreenburg** [comment]
Going to address a few cleanup items in a separate PR

**Pull request merged**